specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/virtualNetworkGateway.json

```diff
@@ -3292,6 +3292,10 @@
           "readOnly": true,
           "type": "string",
           "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "identity": {
+          "$ref": "./network.json#/definitions/ManagedServiceIdentity",
```

Comment:
>./network.json#/definitions/ManagedServiceIdentity [](http://example.com/codeflow?start=19&length=50)

Could you use the managed service identity definition from v5 common types (from managedidentity.json)